Data/Conduit/Algorithms/Async.hs

```diff
@@ -204,10 +205,12 @@ asyncGzipFromFile fname = C.bracketP
 conduitPossiblyCompressedFile :: (MonadBaseControl IO m, MonadResource m) => FilePath -> C.Source m B.ByteString
 conduitPossiblyCompressedFile fname
     | ".gz" `isSuffixOf` fname = asyncGzipFromFile fname
+    | ".xz" `isSuffixOf` fname = C.sourceFile fname =$= CX.decompress oneGBmembuffer
```

Comment:
You probably meant to use `.|` instead of `=$=`

Reply:
I had this code laying around on ngless. It used to compile.
Guess something changed. Included your suggestion. Seems to compile now.